newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -248,10 +259,35 @@ export default class PixiResourcesLoader {
             error ? error : '(unknown error)'
           );
         }
-        loadedCount++;
-        onProgress(loadedCount, imageResources.length);
-      })
-    );
+      }),
+      ...videoResources.map(async resource => {
+        try {
+          const resourceName = resource.getName();
+          const url = ResourcesLoader.getResourceFullUrl(
+            project,
+            resourceName,
+            {
+              isResourceForPixi: true,
+            }
+          );
+
+          loadedTextures[resourceName] = PIXI.Texture.from(url, {
+            scaleMode: PIXI.SCALE_MODES.LINEAR,
+            resourceOptions: {
+              autoPlay: false,
+              crossorigin: determineCrossOrigin(url),
```

Comment:
you can probably use the `determineCrossOrigin` function for the images above too